### DISCUSSION TITLE
Pre-compiled configuration files available?

### DISCUSSION THREAD

**Discussion by yannickgagne** [Jyers/Marlin#1512] [Q&A] [answered]
Hi.

I currently use the BLTouch-HS-10x10-UBL-4.2.2 pre-compiled firmware and it works like a charm, thanks for that!

I was wondering if the configuration files for all the pre-compiled firmwares are available to download because I would like to start from the pre-compiled state and make some changes.

Thanks 😉

**Comment by Jyers** [comment, marked as answer]
All configuration files can be found in the "Configuration Files/E3V2 Templates" folder in the repo
